tests/e2e-test-framework/framework/utils.py

```diff
@@ -508,7 +574,113 @@ def annotate_pvc(
         pvc = self.core_v1_api.read_namespaced_persistent_volume_claim(
             name=resource_name, namespace=namespace
         )
+        if pvc.metadata.annotations is None:
+            pvc.metadata.annotations = {}
         pvc.metadata.annotations[annotation_key] = annotation_value
         self.core_v1_api.patch_namespaced_persistent_volume_claim(
             name=resource_name, namespace=namespace, body=pvc
         )
+
+    def clear_cluster_resources(self, namespace: str) -> None:
```

Comment:
can we change this name to reflect that we delete CSI resources?

Reply:
great idea!